src/schema/artwork/index.js

```diff
@@ -705,6 +706,70 @@ export const artworkFields = () => {
       type: GraphQLString,
       resolve: artwork => (isEmbeddedVideo(artwork) ? null : artwork.website),
     },
+    framed: {
+      type: ArtworkInfoRowType,
+      resolve: ({ framed }) => {
+        if (!framed) {
+          return null
+        }
+        return {
+          label: "Framed",
+          details: "",
+        }
+      },
+    },
+    signatureInfo: {
+      type: ArtworkInfoRowType,
+      resolve: ({
+        signature,
+        signed_by_artist,
+        stamped_by_artist_estate,
+        sticker_label,
+        signed_other,
+      }) => {
+        var detailsParts = []
+        if (signed_by_artist) {
+          detailsParts.push("hand-signed by artist")
+        }
+        if (stamped_by_artist_estate) {
+          detailsParts.push("stamped by artist's estate")
+        }
+        if (sticker_label) {
+          detailsParts.push("sticker label")
+        }
+        if (signature && signature.length > 0) {
+          detailsParts.push(signature)
```

Comment:
If I understand correctly, in the design, the description (free text signature) is after a period, not joined by ", ".

Reply:
Ha... Not sure. Missed that part. Let me check with @briansw